tests/e2e/util.go

```diff
@@ -6823,3 +6826,90 @@ func getVmdkPathFromVolumeHandle(sshClientConfig *ssh.ClientConfig, masterIp str
 	vmdkPath := result.Stdout
 	return vmdkPath
 }
+
+func checkVcServicesHealth(ctx context.Context, host string, timeout ...time.Duration) {
+	var pollTime time.Duration
+	if len(timeout) == 0 {
+		pollTime = pollTimeout * 6
+	} else {
+		pollTime = timeout[0]
```

Comment:
write a logic command, what this if/else is doing and what is the need of this timeout array

Reply:
added